helpdesk_mgmt_ticket_invoice/models/account_move.py

```diff
@@ -0,0 +1,28 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    invoice_ticket_ids = fields.Many2many("helpdesk.ticket", relation="account_helpdesk")
```

Comment:
Field name:
```suggestion
    ticket_ids = fields.Many2many("helpdesk.ticket", relation="account_helpdesk")
```

Other name relation = model_1 + model_2 + rel:
```suggestion
    invoice_ticket_ids = fields.Many2many("helpdesk.ticket", relation="account_move_helpdesk_ticket_rel")
```